assets/blocks/lesson-actions/lesson-actions-block/use-has-quiz.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Has quiz hook.
+ *
+ * @param {Object}   options            Hook options.
+ * @param {Function} options.quizToggle Toggle the quiz block.
+ *
+ * @return {boolean} If a quiz exists with questions.
+ */
+export const useHasQuiz = ( { quizToggle } ) => {
+	const [ quizEventListener ] = useState( null );
```

Comment:
I think we're not using this, right?

Reply:
Yup! Sorry, I was playing around with trying to get the listeners added on first mount. Didn't realize you could just pass an empty array. 102d23bbf5aab52863de9993149a2011508698ca